docs/meshstack.kubernetes.index.md

```diff
@@ -92,6 +79,35 @@ rules:
   - delete
   - update
 - apiGroups:
+  - ""
+  resources:
+  - resourcequotas
+  - resourcequotas/status
+  verbs:
+  - create
+  - delete
+  - deletecollection
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - appliedclusterresourcequotas
+  - clusterresourcequotas
```

Comment:
d: Do we really need the ability to create clusterresourcequotas? This sounds to me like cluster wide quotas that operators can set, but as they are not tenant specific, they are not set by meshStack. Or did i misunderstand these quotas?

Reply:
Actually I am not sure. This was taken over from OpenShift permission set, so I assumed they should be similiar if not the same (a few adaptions needed to be done because of the Groups and Users). And I tested it with this setup and it worked. The replication step uses `V1ResourceQuotaSpec`. So yeah sounds to me that this is a bit unneccessairy too. But I dont find the time now to re-check and rework this whole permission set but I can create a ticket to look into OpenShift and Kubernetes permissions set afterwards.